### PR TITLE
Update securing-your-github-pages-site-with-https.md

### PR DESCRIPTION
**Pull request by evK1**
Added note that Pages site must be public to obtain LE cert. Customer had an issue with this recently and it seems a fairly big oversight not to mention.

### Why:

Closes: 

### Check off the following:

- [ ] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline.

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [ ] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Comment by cmwilson21** [comment]
👋 @evK1 - Thanks so much for opening a PR! I'll get this triaged for review :zap:

File: content/pages/getting-started-with-github-pages/securing-your-github-pages-site-with-https.md [hunk 1]
```diff
@@ -40,6 +40,8 @@ All {% data variables.product.prodname_pages %} sites, including sites that are
 
 When you set or change your custom domain in the Pages settings, an automatic DNS check begins. This check determines if your DNS settings are configured to allow {% data variables.product.prodname_dotcom %} to obtain a certificate automatically. If the check is successful, {% data variables.product.prodname_dotcom %} queues a job to request a TLS certificate from [Let's Encrypt](https://letsencrypt.org/). On receiving a valid certificate, {% data variables.product.prodname_dotcom %} automatically uploads it to the servers that handle TLS termination for Pages. When this process completes successfully, a check mark is displayed beside your custom domain name.
 
+Please note that your GitHub Pages site must be publicly available for a Let's Encrypt certificate to be issued. Once the certificate has been issued you may revert the site to private.
```
Comment (sophietheking) [inline content/pages/getting-started-with-github-pages/securing-your-github-pages-site-with-https.md:43]:
```suggestion
Please note that your {% data variables.product.prodname_pages %} site must be publicly available for a Let's Encrypt certificate to be issued. Once the certificate has been issued you may revert the site to private. 
```

**Review by sophietheking — DISMISSED** [review]
Left one suggestion to add a variable instead of the product name, but this change looks good to me. Thanks for adding this to our Docs!

Feel free to `@mention` me here to get this merged. ✨

**Comment by evK1** [comment]
> Left one suggestion to add a variable instead of the product name, but this change looks good to me. Thanks for adding this to our Docs!
> 
> Feel free to `@mention` me here to get this merged. ✨

@sophietheking - merge away ma'am :)

**Review by sophietheking — APPROVED** [review]

**Pull request merged**